change/@microsoft-fast-foundation-c41dad2f-81e1-49b1-94ef-0b3115cc16d9.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "feat: enable fallback container creation for DI location APIs",
```

Comment:
Should `DI` be spelled out? I'm not sure what it is.

Reply:
I guess it's dependency injection after finishing the review, though I still think we should avoid abbreviating except where using common industry acronyms.  It's common for those in the front-end, but maybe not for other layers or services of the stack.

Reply:
Good point here and below. I think I'm just used to having to keep these under the GitHub character limit for titles. But I guess that doesn't matter for our change files. Need to remember that. I'll update these before merging.